numpy/linalg/linalg.py

```diff
@@ -1915,7 +1915,7 @@ def lstsq(a, b, rcond="warn"):
     x : {(N,), (N, K)} ndarray
         Least-squares solution. If `b` is two-dimensional,
         the solutions are in the `K` columns of `x`.
-    residuals : {(), (1,), (K,)} ndarray
+    residuals : {(0,), (1,), (K,)} ndarray
```

Comment:
I'd just remove the `(0,)` - now it is just confusing and the note states what happens for wrong input.

Reply:
It may be confusing, but that's because the behavior is confusing too!

And it's not even for "wrong input" - just for cases when an exact match is possible. I could move the `(0,)` to the last item in the list.

Reply:
Yes, maybe having it as the last item is best, since it is not the most common output.